task_2/dmitriy.osipov/bookshop/src/main/java/com/task/bookshop/model/Book.java

```diff
@@ -0,0 +1,74 @@
+package com.task.bookshop.model;
+
+import java.util.UUID;
+
+public class Book {
+
+  private static final long serialVersionUID = 170820000000L;
```

Comment:
I've a little hurried up here. 
Findbugs plugin warned about serialVersionUID at my servlets so I had to add this parameters to them (and I discovered HttpServlet implements Serializable). After that I added it to this one for the future changes, but I haven't followed "You ain’t gonna need it" principle.